shared/src/commonMain/kotlin/org/hyperskill/app/placeholder_new_user/injection/PlaceholderNewUserComponentImpl.kt

```diff
@@ -11,6 +12,9 @@ class PlaceholderNewUserComponentImpl(
         get() = PlaceholderNewUserFeatureBuilder.build(
             appGraph.analyticComponent.analyticInteractor,
             appGraph.buildTrackDataComponent().trackInteractor,
+            appGraph.buildProgressesDataComponent().progressesInteractor,
             appGraph.buildProfileDataComponent().profileInteractor
         )
+    override val placeHolderNewUserViewDataMapper: PlaceholderNewUserViewDataMapper
```

Comment:
Add a new line between `placeholderNewUserFeature` and `placeHolderNewUserViewDataMapper`